cdap-docs/tools/github-release-notes.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#  Copyright © 2017 Cask Data, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# GitHub Release Notes
+#
+# Give a CDAP release notes file in rst format, generates a file with release notes for
+# the top-most section in GitHub format (Markdown).
+#
+# Reads and writes using common-sense defaults that can be over-written if required.
+# Default output ends up as 'target/github-release-notes.txt'.
+#
+# It does not resolve references to the CDAP docs; these are flagged
+# 'LINK_TO_BE_CORRECTED' and must be resolved manually.
+#
+# version 0.1
+
+from optparse import OptionParser
+
+import os
+import sys
+
+SCRIPT_DIR_PATH = os.path.dirname(os.path.abspath(__file__))
+DEFAULT_INPUT_RELEASE_NOTES_FILE = os.path.abspath(os.path.join(SCRIPT_DIR_PATH, '../reference-manual/source/release-notes.rst'))
+DEFAULT_OUTPUT_RELEASE_NOTES_FILE = os.path.abspath(os.path.join(SCRIPT_DIR_PATH, '../reference-manual/target/html/github-release-notes.txt'))
```

Comment:
Change these to be more correct. `DEFAULT_INPUT_RELEASE_NOTES_RST_FILE` and `DEFAULT_OUTPUT_RELEASE_NOTES_MD_FILE`

Also, default output location doesn't match comments above.